models/lpjguess/R/write.config.LPJGUESS.R

```diff
@@ -96,16 +96,16 @@ write.config.LPJGUESS <- function(defaults, trait.values, settings, run.id) {
 #' @author Istem Fer
 write.insfile.LPJGUESS <- function(settings, trait.values, rundir, outdir, run.id) {
   
-  guessins  <- readLines(con = system.file("template.ins", package = "PEcAn.LPJGUESS"), n = -1)
-  paramsins <- readLines(con = system.file("pecan.ins", package = "PEcAn.LPJGUESS"), n = -1)
+  guessins  <- base::readLines(con = system.file("template.ins", package = "PEcAn.LPJGUESS"), n = -1)
+  paramsins <- base::readLines(con = system.file("pecan.ins", package = "PEcAn.LPJGUESS"), n = -1)
```

Comment:
Seems odd to need this -- it's usually safe / recommended to use base functions without namespace qualifiers. Is there a competing non-base version of readLines that this is avoiding?

Reply:
at some point I thought we decided to use namespace even for the base functions, but looks like it was a misunderstanding on my end. Will just revert these